autogpt_platform/frontend/src/app/(platform)/profile/(user)/integrations/page.tsx

```diff
@@ -5,6 +5,7 @@ import { useCallback, useContext, useEffect, useMemo, useState } from "react";
 import { useToast } from "@/components/ui/use-toast";
 import { IconKey, IconUser } from "@/components/ui/icons";
 import { Trash2Icon } from "lucide-react";
+import { FaKey } from "react-icons/fa";
```

Comment:
I have agreed with Olivia to settle on **Phospor icons**, so we don't have like 5 icons libraries on the app 😂 

This one should be good: https://phosphoricons.com/?q=key

```tsx
import { KeyIcon } from ""@phosphor-icons/react/dist/ssr"
```